src/pseudopeople/noise_entities.py

```diff
@@ -10,63 +9,81 @@ class __NoiseTypes(NamedTuple):
     omissions, duplications, missing data, incorrect selection, copy from w/in
     household, month and day swaps, zip code miswriting, age miswriting,
     numeric miswriting, nicknames, fake names, phonetic, OCR, typographic"
+
+    NOTE: Any configuration tree overwrites in these objects are what ends up
+    in the "baseline" ConfigTree layer.
     """
 
-    # todo finish filling in noise types in the correct order per the docs
-    OMISSION: RowNoiseType = RowNoiseType("omission", noise_functions.omit_rows)
-    DUPLICATION: RowNoiseType = RowNoiseType("duplication", noise_functions.duplicate_rows)
-    MISSING_DATA: ColumnNoiseType = ColumnNoiseType(
+    omission: RowNoiseType = RowNoiseType(
+        "omission", noise_functions.omit_rows, is_implemented=False
```

Comment:
Set this to `is_implemented=True` even though it isn't so the row_noise block appears.

Reply:
Uh oh - I've removed all of the is_implemented stuff as discussed yesterday. But yes, I've kept omission even though it's not doing anything.